app/models/service/retirement_management.rb

```diff
@@ -7,6 +7,7 @@ def before_retirement
   end
 
   def retire_service_resources
+    return
```

Comment:
@d-m-u Add a TODO here so we can delete the code at some point or mark it as experimental.